bin/node-pg-migrate

```diff
@@ -7,9 +7,7 @@ const util = require('util')
 const path = require('path')
 const yargs = require('yargs')
 const ConnectionParameters = require('pg/lib/connection-parameters')
-const migrationRunner = require('../index')
-
-const { Migration } = migrationRunner
+const { default: migrationRunner, Migration } = require('../dist')
```

Comment:
Cool ✨ 
Didn't know that this is possible

Reply:
The problem is, that typescript does not produce commonjs compatible default export.
https://github.com/Microsoft/TypeScript/issues/2719
But as the es6 is future 🙏  and many other packages switched to using es6 exports/imports, I will not try to fix it...